dallinger/frontend/static/scripts/dallinger2.js

```diff
@@ -307,34 +392,119 @@ var dallinger = (function () {
     return deferred;
   };
 
+  /**
+   * Creates a new experiment node for the current partcipant.
+   *
+   * @example
+   * var response = dallinger.createAgent();
+   * // Wait for response
+   * response.done(function (data) {... handle data.node ...});
+   *
+   * @returns {jQuery.Deferred} See :ref:`deferreds-label`
+   */
   dlgr.createAgent = function () {
     return dlgr.post('/node/' + dallinger.identity.participantId);
   };
 
+  /**
+   * Creates a new info object in the experiment database.
+   *
+   * @example
+   * var response = dallinger.createInfo(1, {details: {a: 1}});
+   * // Wait for response
+   * response.done(function (data) {... handle data.info ...});
+   *
+   * @param {number} nodeId - The id of the participant's experiment node
+   * @param {Object} data - Experimental data (see :class:`~dallinger.models.Info`)
+   * @returns {jQuery.Deferred} See :ref:`deferreds-label`
+   */
   dlgr.createInfo = function (nodeId, data) {
     return dlgr.post('/info/' + nodeId, data);
   };
 
+  /**
+   * Returns a public property value for the experiment.
+   *
+   * @example
+   * var response = dallinger.getExperimentProperty("propname");
+   * // Wait for response
+   * response.done(function (data) {... handle e.g. data.propname ...});
+   *
+   * @param {string} prop - The experiment property to lookup
+   * @returns {jQuery.Deferred} See :ref:`deferreds-label`
+   */
   dlgr.getExperimentProperty = function (prop) {
     return dlgr.get('/experiment/' + prop);
   };
 
+  /**
+   * Get a specific info object.
```

Comment:
Capitalize types? ("Info") I noticed it's capitalized below in the formal docstring... maybe that's your convention (informal in narrative, capitalized class names in docstring)? Also wondering about capitalizing "ID", which we seem to do elsewhere (though perhaps not consistently).